### ISSUE TITLE
Support macOS Mojave dark mode

### ISSUE THREAD

**Issue by jsynowiec** [jsynowiec/airqmon#35]
Apple released [design resources for macOS](https://developer.apple.com/design/resources): [Introducing Apple Design Resources for macOS](https://developer.apple.com/design/whats-new/?id=07252018)
There are also videos from WWDC, e.g. ["Introducing Dark Mode"](https://developer.apple.com/videos/play/wwdc2018/210/) and [guidelines](https://developer.apple.com/design/human-interface-guidelines/macos/visual-design/dark-mode/), [AppKit Article - Supporting Dark Mode in Your Interface](https://developer.apple.com/documentation/appkit/supporting_dark_mode_in_your_interface).

Blocked by: https://github.com/electron/electron/pull/14755

**Comment by DCzajkowski** [comment]
Isn't this enough?
```js
const { systemPreferences: { isDarkMode } } = require('electron')
if (isDarkMode()) {
  // set dark theme
} else {
  // set light theme
}
```

**Comment by jsynowiec** [comment]
As a temporary solution, maybe.

But I to do it properly, application should support [NSApplication.appearance](https://developer.apple.com/documentation/appkit/nsapplication/2967170-appearance). And this is what https://github.com/electron/electron/pull/14755 adds.

Nevertheless, this has a low priority for me currently. There is no support for proper _theming_ in the app right now and I'm focused on other parts.

**Comment by jsynowiec** [comment]
This is how it'll look in the next version. Light and dark appearance. It changes in response to the system settings so if one uses F.lux or some other app to automatically change between appearances on sunset, then Airqmon adapts in background. Vibrancy is also supported.

<img width="206" alt="Screenshot 2019-03-26 at 11 14 38" src="https://user-images.githubusercontent.com/1029142/54989495-f4bb4080-4fb8-11e9-8f8b-85d74db5bf13.png"><img width="206" alt="Screenshot 2019-03-26 at 11 14 00" src="https://user-images.githubusercontent.com/1029142/54989497-f4bb4080-4fb8-11e9-9a3c-bcaa5b38066d.png">

This might interest you @thymikee.
